src/Action/action_lib/action_u1_dm.jl

```diff
@@ -210,6 +210,84 @@ function symmetrize_add_to!(
+                a.Γ[6].ch_s.q2_2[i, w, v] += a_l.Γ[6].ch_s.q2_2[i, w, v] - a_l.Γ[5].ch_s.q2_1[r.exchange[i], w, v]
+
+                # add q2_1 and q2_2 to t channel (right part from v <-> v' exchange)
+                a.Γ[1].ch_t.q2_1[i, w, v] += a_l.Γ[1].ch_t.q2_1[i, w, v] + a_l.Γ[1].ch_t.q2_2[r.exchange[i], w, v]
+                a.Γ[2].ch_t.q2_1[i, w, v] += a_l.Γ[2].ch_t.q2_1[i, w, v] + a_l.Γ[2].ch_t.q2_2[r.exchange[i], w, v]
+                a.Γ[3].ch_t.q2_1[i, w, v] += a_l.Γ[3].ch_t.q2_1[i, w, v] - a_l.Γ[3].ch_t.q2_2[r.exchange[i], w, v]
+                a.Γ[4].ch_t.q2_1[i, w, v] += a_l.Γ[4].ch_t.q2_1[i, w, v] + a_l.Γ[4].ch_t.q2_2[r.exchange[i], w, v]
+                a.Γ[5].ch_t.q2_1[i, w, v] += a_l.Γ[5].ch_t.q2_1[i, w, v] - a_l.Γ[6].ch_t.q2_2[r.exchange[i], w, v]
+                a.Γ[6].ch_t.q2_1[i, w, v] += a_l.Γ[6].ch_t.q2_1[i, w, v] - a_l.Γ[5].ch_t.q2_2[r.exchange[i], w, v]
+                a.Γ[1].ch_t.q2_2[i, w, v] += a_l.Γ[1].ch_t.q2_2[i, w, v] + a_l.Γ[1].ch_t.q2_1[r.exchange[i], w, v]
+                a.Γ[2].ch_t.q2_2[i, w, v] += a_l.Γ[2].ch_t.q2_2[i, w, v] + a_l.Γ[2].ch_t.q2_1[r.exchange[i], w, v]
+                a.Γ[3].ch_t.q2_2[i, w, v] += a_l.Γ[3].ch_t.q2_2[i, w, v] - a_l.Γ[3].ch_t.q2_1[r.exchange[i], w, v]
+                a.Γ[4].ch_t.q2_2[i, w, v] += a_l.Γ[4].ch_t.q2_2[i, w, v] + a_l.Γ[4].ch_t.q2_1[r.exchange[i], w, v]
+                a.Γ[5].ch_t.q2_2[i, w, v] += a_l.Γ[5].ch_t.q2_2[i, w, v] - a_l.Γ[6].ch_t.q2_1[r.exchange[i], w, v]
+                a.Γ[6].ch_t.q2_2[i, w, v] += a_l.Γ[6].ch_t.q2_2[i, w, v] - a_l.Γ[5].ch_t.q2_1[r.exchange[i], w, v]
+
+                # add q2_1 and q2_2 to u channel (right part from v <-> v' exchange)
+                a.Γ[1].ch_u.q2_1[i, w, v] += a_l.Γ[1].ch_u.q2_1[i, w, v] + a_l.Γ[1].ch_u.q2_2[i, w, v]
+                a.Γ[2].ch_u.q2_1[i, w, v] += a_l.Γ[2].ch_u.q2_1[i, w, v] + a_l.Γ[2].ch_u.q2_2[i, w, v]
+                a.Γ[3].ch_u.q2_1[i, w, v] += a_l.Γ[3].ch_u.q2_1[i, w, v] + a_l.Γ[3].ch_u.q2_2[i, w, v]
+                a.Γ[4].ch_u.q2_1[i, w, v] += a_l.Γ[4].ch_u.q2_1[i, w, v] + a_l.Γ[4].ch_u.q2_2[i, w, v]
+                a.Γ[5].ch_u.q2_1[i, w, v] += a_l.Γ[5].ch_u.q2_1[i, w, v] - a_l.Γ[5].ch_u.q2_2[i, w, v]
+                a.Γ[6].ch_u.q2_1[i, w, v] += a_l.Γ[6].ch_u.q2_1[i, w, v] - a_l.Γ[6].ch_u.q2_2[i, w, v]
+                a.Γ[1].ch_u.q2_2[i, w, v] += a_l.Γ[1].ch_u.q2_2[i, w, v] + a_l.Γ[1].ch_u.q2_1[i, w, v]
+                a.Γ[2].ch_u.q2_2[i, w, v] += a_l.Γ[2].ch_u.q2_2[i, w, v] + a_l.Γ[2].ch_u.q2_1[i, w, v]
+                a.Γ[3].ch_u.q2_2[i, w, v] += a_l.Γ[3].ch_u.q2_2[i, w, v] + a_l.Γ[3].ch_u.q2_1[i, w, v]
+                a.Γ[4].ch_u.q2_2[i, w, v] += a_l.Γ[4].ch_u.q2_2[i, w, v] + a_l.Γ[4].ch_u.q2_1[i, w, v]
+                a.Γ[5].ch_u.q2_2[i, w, v] += a_l.Γ[5].ch_u.q2_2[i, w, v] - a_l.Γ[5].ch_u.q2_1[i, w, v]
+                a.Γ[6].ch_u.q2_2[i, w, v] += a_l.Γ[6].ch_u.q2_2[i, w, v] - a_l.Γ[6].ch_u.q2_1[i, w, v]
```

Comment:
Just for me: is there an easy way to see, that zd does not become dz when exchanging v and v'?

Reply:
I don't think there is an easy way to see this. A short explanation would be:
The v <-> v' symmetry originates from either the s -> -s, t -> -t, or u -> -u symmetry, depending on the channel. In the u-channel it originates from the t -> -t symmetry, which unlike the s -> -s and the u -> -u symmetry, does not come with a site and spin exchange (see Finn's paper). Therefore there is also no mapping of dz to zd and xy -> yx.